### DISCUSSION TITLE
Restarting karavan

### DISCUSSION THREAD

**Discussion by bitsInnovate00** [apache/camel-karavan#1053] [Q&A] [answered]
What is the general guideline on restarting karavan in kubernetes installation ? For eg: If I need to change the container registry and restart karavan , which all kubernetes resources I need to delete and start again. Or do I need to delete the karavan namespace completely and execute the command
java -jar karavan-installer-4.3.0.jar

**Comment by mgubaidullin** [comment, marked as answer]
Container registry used by devmode container when build integration microservice and by kubernetes when microservice deployed.
You do not need to restart karavan when change container registry
